src/test/java/TypewiseAlert/TypewiseAlertTest.java

```diff
@@ -1,15 +1,139 @@
 package TypewiseAlert;
 
-import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertEquals;
 
 import org.junit.Test;
 
-public class TypewiseAlertTest 
+public class TypewiseAlertTest
```

Comment:
duplication code found in most of the test cases ...sanitize the same using setup() teardown() methods